bokehjs/src/coffee/models/transforms/jitter.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require "underscore"
+Transform = require "./transform"
+p = require "../../core/properties"
+
+class Jitter extends Transform.Model
+  initialize: (attrs, options) ->
+    super(attrs, options)
+
+  props: ->
+    return _.extend {}, super(), {
+      mean:         [ p.Number, 0        ]
+      width:        [ p.Number, 1        ]
+      distribution: [ p.String, 'uniform']
```

Comment:
Corrected.

As a side note.  I think it would have been nice if I could have specified this as:

    [p.Enum(enums.DistributionType), 'uniform']

It seems a bit dangerous for people to need to edit the important properties.coffee file to add an enum type.  I suppose I could have just put:

    [p.enum_prop('Distribution', enums.DistributionType), 'uniform']

I think the former is cleaner looking, but it still requires the user to pull in two files instead of just the properties...  Who knows which is better...